grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -254,7 +255,7 @@ class InvoiceService {
         invoice.description = order.description
         invoice.partyFrom = order.destinationParty
         invoice.party = order.origin.organization
-        invoice.dateInvoiced = new Date()
+        invoice.dateInvoiced = LocalDate.now().toDate()
```

Comment:
What is wrong with the previous version?

Reply:
It set the date with time, filter is searching for the date with no time set so it didn't match.

Reply:
I think we're ok with keeping the time.